### PR TITLE
chore: adds angular 16 test app

### PR DESCRIPTION
**Pull request by jaredperreault-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](/okta/okta-angular/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: scripts/e2e-v6.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ export ISSUER=https://samples-javascript.okta.com/oauth2/default
 export SPA_CLIENT_ID=0oapmwm72082GXal14x6
 export USERNAME=george@acme.com
 get_vault_secret_key devex/samples-javascript password PASSWORD
-export ORG_OIE_ENABLED=
+# export ORG_OIE_ENABLED=
```
Comment (shuowu-okta) [inline scripts/e2e-v6.sh:20]: remove?

File: scripts/e2e.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ export ISSUER=https://samples-javascript.okta.com/oauth2/default
 export SPA_CLIENT_ID=0oapmwm72082GXal14x6
 export USERNAME=george@acme.com
 get_vault_secret_key devex/samples-javascript password PASSWORD
-export ORG_OIE_ENABLED=
+# export ORG_OIE_ENABLED=
```
Comment (shuowu-okta) [inline scripts/e2e.sh:15]: remove?

File: scripts/setup-e2e.sh [hunk 3]
```diff
@@ -34,7 +34,8 @@ install_auth_js () {
 
 # Install required node version
 export NVM_DIR="/root/.nvm"
-setup_service node v16.13.2
+setup_service node v16.14.0
+setup_service python 3.6
```
Comment (shuowu-okta) [inline scripts/setup-e2e.sh:38]: just curious, why python will be needed?

**Review by shuowu-okta — APPROVED** [review]

File: scripts/setup-e2e.sh (hunk 3 above)
Comment (jaredperreault-okta) [reply scripts/setup-e2e.sh:38]: Some node-gyp scripts during angular installations were failing in CI, the error message didn't like the python version installed

**Pull request merged**